Casks/iojones.rb

```diff
@@ -0,0 +1,10 @@
+cask 'iojones' do
+  version '1.1'
+  sha256 '381cd0f643c89f016e30cf978d361dcf06c396d6d1ed76e9ae9c192a3ed060b7'
+
+  url "https://jaist.dl.sourceforge.net/project/iojones/IOJones/1.1/IOJones.zip"
```

Comment:
```suggestion
  url "https://jaist.dl.sourceforge.net/project/iojones/IOJones/#{version}/IOJones.zip"
```

This is why you need to follow the template. Doing so would have caught this error.

Reply:
Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Reply:
added